app/src/main/java/ch/sdp/vibester/activity/ScoreBoardActivity.kt

```diff
@@ -26,7 +30,26 @@ class ScoreBoardActivity : AppCompatActivity() {
         setSupportActionBar(findViewById(R.id.toolbar))
         setupRecycleView()
         players = ArrayList()
-        loadPlayers()
+    }
+
+    /**
+     * TODO: replace "ranking" by appropriate label
+     */
+    fun selectScoreboard(view: View) {
+        var sortedBy = ""
+        when (view.id) {
+            R.id.btsButton -> sortedBy = "ranking"
+            R.id.kpopButton -> sortedBy = "ranking"
+            R.id.imagDragonsButton -> sortedBy = "ranking"
+            R.id.billieEilishButton -> sortedBy = "ranking"
+            R.id.rockButton -> sortedBy = "ranking"
+            R.id.topTracksButton -> sortedBy = "ranking"
```

Comment:
```suggestion
            R.id.btsButton -> sortedBy = "ranking"
            R.id.kpopButton -> sortedBy = "ranking"
            R.id.imagDragonsButton -> sortedBy = "ranking"
            R.id.billieEilishButton -> sortedBy = "ranking"
            R.id.rockButton -> sortedBy = "ranking"
            R.id.topTracksButton -> sortedBy = "ranking"
```
Could store "ranking" into string.xml ? easier to modify after